storm-client/test/jvm/org/apache/storm/grouping/LoadAwareShuffleGroupingTest.java

```diff
@@ -204,24 +204,21 @@ private void testLoadAwareShuffleGroupingWithEvenLoadMultiThreaded(int numTasks)
 
         List<Callable<int[]>> threadTasks = Lists.newArrayList();
         for (int x = 0; x < numThreads; x++) {
-            Callable<int[]> threadTask = new Callable<int[]>() {
-                @Override
-                public int[] call() throws Exception {
-                    int[] taskCounts = new int[availableTaskIds.size()];
-                    for (int i = 1; i <= groupingExecutionsPerThread; i++) {
-                        List<Integer> taskIds = grouper.chooseTasks(inputTaskId, Lists.newArrayList());
-
-                        // Validate a single task id return
-                        assertNotNull("Not null taskId list returned", taskIds);
-                        assertEquals("Single task Id returned", 1, taskIds.size());
-
-                        int taskId = taskIds.get(0);
-
-                        assertTrue("TaskId should exist", taskId >= 0 && taskId < availableTaskIds.size());
-                        taskCounts[taskId]++;
-                    }
-                    return taskCounts;
+            Callable<int[]> threadTask = () -> {
+                int[] taskCounts = new int[availableTaskIds.size()];
+                for (int i = 1; i <= groupingExecutionsPerThread; i++) {
+                    List<Integer> taskIds = grouper.chooseTasks(inputTaskId, Lists.newArrayList());
+
+                    // Validate a single task id return
+                    assertNotNull(taskIds, "Not null taskId list returned");
+                    assertEquals(1, taskIds.size(), "Single task Id returned");
```

Comment:
error message should be negated, example: "Single taskId expected, found " + taskIds.size() + " taskIds

Reply:
done